locales/ja.json

```diff
@@ -1,14 +1,14 @@
 {
   "Execute binaries from npm packages.\n%s": "npmパッケージのバイナリを実行します。",
   "Package to be installed.": "インストールするパッケージ。",
-  "Location of the npm cache.": "npmキャッシュへのパス。",
+  "Location of the npm cache.": "npmのキャッシュへのパス。",
   "Skip installation if a package is missing.": "パッケージがない場合は、インストールをスキップします。",
   "Path to user npmrc.": "ユーザーのnpmrcへのパス。",
-  "Execute string as if inside `npm run-script`.": "「npm run-script」の内部にあるかのように文字列を実行する。",
+  "Execute string as if inside `npm run-script`.": "「npm run-script」の内部にあるかのように文字列を実行します。",
   "Shell to execute the command with, if any.": "コマンドを実行するシェル（存在する場合）。",
-  "Generate shell code to use npx as the \"command not found\" fallback.": "「コマンドが見つかりません」フォールバックとして使用するコードを生成します。",
-  "Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.": "$PATHまたはローカルプロジェクトの既存のバイナリを無視します。 これにより、npxは一時的なインストールを行い、最新バージョンを使用します。",
-  "npm binary to use for internal operations.": "npmバイナリを内部操作に使用します。",
+  "Generate shell code to use npx as the \"command not found\" fallback.": "コマンドが見つかりません。フォールバックとして使用するコードを生成します。",
```

Comment:
That's a good point! `「」` is basically used in a talk between humans([wikipedia in ja](https://ja.wikipedia.org/wiki/%E6%8B%AC%E5%BC%A7#.E9.89.A4.E6.8B.AC.E5.BC.A7.E3.80.8C.E3.80.8D)). We might use double quotes as is, here is an example:
+ https://github.com/vim-jp/lang-ja/blob/master/src/po/ja.po

I will add double quotes if it's fine :)